src/order/rel_classes.lean

```diff
@@ -239,10 +239,20 @@ theorem well_founded.asymmetric {α : Sort*} {r : α → α → Prop} (h : well_
 using_well_founded { rel_tac := λ _ _, `[exact ⟨_, h⟩],
                      dec_tac := tactic.assumption }
 
+theorem well_founded.asymmetric₃ {α : Sort*} {r : α → α → Prop} (h : well_founded r) :
```

Comment:
This name feels quite odd to me; is there precedence elsewhere in mathlib?

Reply:
I don't believe so. This is basically the three variable analog of asymmetry though.

Reply:
What do you think of `asymmetric_transitive` or similar? Either way, I think a docstring would help with explaining the weird name.

Reply:
Hmm... I don't know. The name `asymmetric₃` is surprising at first but does make to sense. This not transitivity, so `asymmetric_transitive` is inadequate. It's more like antitransitivity (does that make it what antisymmmetry is to symmetry, somehow?). Whichever name you land on, please do add a docstring explaining it.

Reply:
Docstring has been added.